doc/aptinstall.sh

```diff
@@ -12,6 +12,7 @@
 
 set -o errexit
 
+sudo apt update
```

Comment:
Is this a change you are introducing or is it to match a change that's been made upstream? If so, can you please explain why it's needed? It feels a bit out of place to me that this PR is also changing something so far removed.

Reply:
Fixes #24098 (should probably have been moved in a separate PR)